src/webview/mod.rs

```diff
@@ -308,7 +308,7 @@ impl<'a> WebViewBuilder<'a> {
   /// # Warning
   /// This will call private functions on **macOS**. It's still enabled if set in **debug** build on mac,
   /// but requires `devtool` feature flag to actually enable it in **release** build.
```

Comment:
```suggestion
  /// but requires `devtools` feature flag to actually enable it in **release** build.
```
And also changing the actual feature flag name.